pkg/controller/hostpathprovisioner/daemonset.go

```diff
@@ -176,15 +174,30 @@ func createDaemonSetObject(cr *hostpathprovisionerv1.HostPathProvisioner, reqLog
 									Name: "NODE_NAME",
 									ValueFrom: &corev1.EnvVarSource{
 										FieldRef: &corev1.ObjectFieldSelector{
-											APIVersion: "v1",
-											FieldPath:  "spec.nodeName",
+											FieldPath: "spec.nodeName",
 										},
 									},
 								},
 								{
 									Name:  "PV_DIR",
 									Value: cr.Spec.PathConfig.Path,
 								},
+								{
+									Name: "INSTALLER_PART_OF_LABEL",
+									ValueFrom: &corev1.EnvVarSource{
+										FieldRef: &corev1.ObjectFieldSelector{
+											FieldPath: "metadata.labels['app.kubernetes.io/part-of']",
```

Comment:
Same here, we want to add APIVersion field so the operator doesn't attempt to reconcile it away.

Reply:
Done